models/webhooks/_all.yml

```diff
@@ -180,7 +180,7 @@ properties:
     allOf:
       - $ref: ./store_customer_payment_instrument_default_updated.yml
   store/customer/updated:
-    description: Fires when a customer is updated. Does not currently track changes to the customer address.
+    description: Fires when a customer is updated. Does not currently track changes to the customer address. Tracks changes to customer attributes only if the changes are made using the Control Panel.

```

Comment:
FYI: https://github.com/bigcommerce/dev-docs/pull/2042/files#r1289439695

Reply:
👍 Updated the description based on your feedback